tests/Adapters/Microsoft.Bot.Builder.Adapters.Twilio.Tests/TwilioAdapterTests.cs

```diff
@@ -16,57 +16,28 @@ namespace Microsoft.Bot.Builder.Adapters.Twilio.Tests
     public class TwilioAdapterTests
     {
         [Fact]
-        public void Constructor_Should_Fail_With_Null_Options()
-        {
-            Assert.Throws<ArgumentNullException>(() => { new TwilioAdapter(null, new Mock<TwilioClientWrapper>().Object); });
-        }
-
-        [Fact]
-        public void Constructor_Should_Fail_With_Null_TwilioNumber()
-        {
-            var options = new TwilioAdapterOptions(null, "Test", "Test", new Uri("http://contoso.com"));
-
-            Assert.Throws<ArgumentException>(() => { new TwilioAdapter(options, new Mock<TwilioClientWrapper>().Object); });
-        }
-
-        [Fact]
-        public void Constructor_Should_Fail_With_Null_AccountSid()
-        {
-            var options = new TwilioAdapterOptions("Test", null, "Test", new Uri("http://contoso.com"));
-
-            Assert.Throws<ArgumentException>(() => { new TwilioAdapter(options, new Mock<TwilioClientWrapper>().Object); });
-        }
-
-        [Fact]
-        public void Constructor_Should_Fail_With_Null_AuthToken()
-        {
-            var options = new TwilioAdapterOptions("Test", "Test", null, new Uri("http://contoso.com"));
-
-            Assert.Throws<ArgumentException>(() => { new TwilioAdapter(options, new Mock<TwilioClientWrapper>().Object); });
-        }
-
-        [Fact]
-        public void Constructor_Should_Fail_With_Null_TwilioApi()
+        public void Constructor_Should_Fail_With_Null_TwilioAPIWrapper()
```

Comment:
Is it okay for the name of the tests to have underscores?

Reply:
true. It's not